evergreen.yml

```diff
@@ -68,21 +68,24 @@ functions:
             export RELEASE_VERSION=snapshot
             export LIBMONGOTRANSLATE_VERSION=snapshot
           fi
-          export MSI_FILENAME="mongoodbc.msi"
+          export MSI_FILENAME="mongoodbc-eap-$RELEASE_VERSION-lib-$LIBMONGOTRANSLATE_VERSION.msi"
```

Comment:
I'm wondering if need to go this granular on the file name.
It make sense because we can technically have vx.y.z-libva.b.c and vx.y.z-libvd.e.f tags, but at the same time, it is not our intent. An upgrade of mongotranslate means a new version of ODBC is needed to bundle it so we should always have a different ODBC version.
 and for users, it would be much easier to only have to deal with the OBDC driver version. They shouldn't have to worry about the mongotranslate library version since we take care of everything for them.
Our release directory path has the mongotranslate version info so we shouldn't have any mix up problem if we mess up a tag (even if it should not happen).

Reply:
That's cool with me too; I'm happy to remove the lib version from the name.